src/pybind/mgr/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ if(WITH_MGR_ROOK_CLIENT)
 endif()
 if(WITH_TESTS)
   include(AddCephTest)
-  add_tox_test(mgr ${CMAKE_CURRENT_SOURCE_DIR} TOX_ENVS py3 py37 mypy flake8 jinjalint nooptional)
+  add_tox_test(mgr ${CMAKE_CURRENT_SOURCE_DIR} TOX_ENVS py3 python3.7 mypy flake8 jinjalint nooptional)
```

Comment:
i don't understand what these changes are doing. can we find someone familiar with python and ceph-mgr to review this part? maybe @epuertat @leseb?

Reply:
jenkins test make check

Reply:
Without this commit there is an error in the make check:
  py3: commands succeeded
SKIPPED:  py37: InterpreterNotFound: python3.7
ERROR:   mypy: commands failed

Anyway it's not mandatory to add this commit to the PR so once the make check is fine I'll drop it